tests/pytests/conftest.py

```diff
@@ -547,6 +548,21 @@ def ssl_webserver(integration_files_dir, this_txt_file):
         yield webserver
 
 
+@pytest.fixture(scope="module")
+def website_contents(integration_files_dir):
+    with tarfile.open(integration_files_dir / "test.tar.gz", "w:gz") as tar:
```

Comment:
Where is this getting used?

Reply:
In the webserver pytest fixture, I guess I could just add that directly into the webserver fixture itself, or add it in the test since its related to just the test.

Reply:
My question was more like, I see the `.txt` being requested, but not the compressed file.

Or am I wrong?

Reply:
This is where the compressed file is being requested: https://github.com/saltstack/salt/pull/64992/files#diff-aa7902e22e8ffb79a4b957472f29be727a98ac9d644ef619688394e1fb4a698eR14